evap/staff/views.py

```diff
@@ -176,6 +176,7 @@ def semester_course_operation(request, semester_id):
         operation=operation,
         current_state_name=current_state_name,
         new_state_name=new_state_name,
+        no_email_btn=operation in ['prepare', 'reenableEditorReview', 'publish']
```

Comment:
the button is a checkbox now, could you call that `show_email_checkbox` or something?